Pipfile

```diff
@@ -64,7 +64,6 @@ django-static-jquery = "==2.1.4"
 setuptools = "*"
 faker = "*"
 werkzeug = "*"
-black = "==19.10b0"
```

Comment:
you could also bin flake8 here, but can also be done under https://github.com/DemocracyClub/electionleaflets/issues/267

Reply:
Done in [5ef4ce1](https://github.com/DemocracyClub/electionleaflets/pull/285/commits/5ef4ce127f64533fa9fcaae3665ff5121ecd4505)